stats/lib/platform-routes.js

```diff
@@ -39,7 +39,9 @@ export const handlePlatformRoutes = async (req, res, pgPools) => {
   } else if (req.method === 'GET' && url === '/participants/top-earning') {
     await respond(pgPools.stats, fetchTopEarningParticipants)
   } else if (req.method === 'GET' && url === '/participants/summary') {
-    json(res, await fetchParticipantsSummary(pgPools.evaluate))
+    const summary = await fetchParticipantsSummary(pgPools.evaluate)
+    res.setHeader('cache-control', `public, max-age=${24 * 3600 /* one day */}`)
+    json(res, summary)
```

Comment:
Would you mind extracting these three lines into a new function? I would like to keep `handlePlatformRoutes` at a single level of abstraction - match request method+path and call the appropriate route handler.

Reply:
I have refactored dc18234 but don't know where to put it. I also don't see a good way to refactor a generic `getStatsWithCaching` function. Can you please advise in more detail how you envision this?